Makefile

```diff
@@ -328,7 +328,11 @@ mango-test: devclean all
 	@cd src/mango && \
 		python3 -m venv .venv && \
 		.venv/bin/python3 -m pip install -r requirements.txt
-	@cd src/mango && ../../dev/run "$(TEST_OPTS)" -n 1 --admin=testuser:testpass '.venv/bin/python3 -m nose2'
+	@cd src/mango && \
```

Comment:
Need Makefile.win the same change?

Reply:
Oh, yes, you are right.  I forgot about the Windows bit.  I will add them soon.

Reply:
I think this is covered now.